frame/nfts/src/features/approvals.rs

```diff
@@ -100,6 +136,19 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		Ok(())
 	}
 
+	/// Clears all transfer approvals for an item.
+	///
+	/// This function is used to clear all transfer approvals for the specified `item` in the
+	/// `collection`. If `maybe_check_origin` is specified, the function ensures that the
+	/// `check_origin` account is the owner of the item, granting permission to clear all transfer
+	/// approvals. After clearing all approvals, the function emits the `AllApprovalsCancelled`
+	/// event.
+	///
+	/// - `maybe_check_origin`: The optional account that is required to be the owner of the item,
+	///   granting
+	/// permission to clear all transfer approvals. If `None`, no permission check is performed.
```

Comment:
Those multi-line comments got broke in a weird way

Reply:
Thanks for fixing!